recipes/openssl/1.x.x/conanfile.py

```diff
@@ -758,9 +758,57 @@ def package(self):
 
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_subfolder, self._module_file)
+        )
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file):
+        content = (
+            "if(OpenSSL_FOUND)\n"
```

Comment:
Let's have this as a separate file in the repo if it's possible?

Reply:
I don't know, I don't want to see the content of this file too far from package_info because it's tightly related.

Reply:
If you want to keep it embedded maybe use a multi line string (`"""`)?

Reply:
I don't like multi lines because it breaks indentation and looks ugly in a python file.

Reply:
Taste is different I guess :)
I actually prefer the zero indentation but for those who like to keep it I have seen this pattern used:
```py
import textwrap

def foo():
    content = """\
        for
        bar
        baz
    """
    print(textwrap.dedent(content))

foo()
```
This is of course not an important topic, just want to share the pattern.

Reply:
It does not produce the same string. You'll have plenty of unwanted spaces.

Reply:
changed in https://github.com/conan-io/conan-center-index/pull/4597/commits/d5cb67db94c31a65737d2a30ce819608b4014141, indeed it's neater ;)